scripts/core/EXT.rst

```diff
@@ -14,25 +18,42 @@ Objective
 =========
 
 Features which are device- or vendor-specific can be exposed as extensions.
-The list of extensions supported by the driver implementation can be queried using ${x}DriverGetExtensionProperties.
 
 There are two types of extensions defined by this specification:
 
 1. **Standard** - extensions ratified for inclusion into the current and all future versions of the specification.
 2. **Experimental** - extensions require additional experimentation and feedback from application vendors
    before ratification, therefore applications should not rely on experimental extensions in production.
 
-In addition, an implementation may choose to provide additional non-standard extensions via ${x}DriverGetExtensionFunctionAddress.
-It is the responsibility of the implementation to ensure these extensions do not conflict or interfere with any features or extensions of the core APIs.
-These extensions are not documented as part of this specification.
+In addition, an implementation may choose to provide additional non-standard extensions 
+It is the responsibility of the implementation to ensure these extensions do not conflict or interfere with any features or extensions of the standard 
+core or sysman APIs. These extensions are not documented as part of this specification.
+
+The following table summarizes the APIs to be used for discovery of different types of extensions.
+
++----------------------------------+-----------------------------+---------------------------------------+
+| API Category                     | Extension Type              | Extension Discovery API               |
++==================================+=============================+=======================================+
+| Core                             | Standard                    | ${x}DriverGetExtensionProperties      |
+|                                  +-----------------------------|                                       |
+|                                  | Experimental                |                                       |
++----------------------------------+-----------------------------+---------------------------------------+
+| Core                             | Implementation non-standard | ${x}DriverGetExtensionFunctionAddress |
```

Comment:
"implementation non-standard, you mean zeDriverGetExtensionFunctionAddress or the function which pointer you get with zeDriverGetExtensionFunctionAddress?

Reply:
implementation non-standard = driver experimental. just re-using existing spec terminology here since the wording is "an implementation may choose to provide additional non-standard extensions"

Reply:
thanks @aravindksg . but you you are not referring to zeDriverGetExtensionFunctionAddress , since zeDriverGetExtensionFunctionAddress  it is standard. What is not standard is the function to which you are getting access with zeDriverGetExtensionFunctionAddress , right?

Reply:
yes, as mentioned prior to this table, this is about the discovery mechanism : "following table summarizes the APIs to be used for discovery of different types of extensions"